src/js/entities-service/entities/groups.js

```diff
@@ -11,6 +13,45 @@ const _Model = BaseModel.extend({
   getClinicians() {
     return Radio.request('entities', 'clinicians:collection', this.get('_clinicians'));
   },
+  addClinician(clinician) {
+    const url = `/api/groups/${ this.id }/relationships/clinicians`;
+    const group = this.toRelation(this);
+
+    clinician.set({ _groups: clinician.get('_groups').concat({ id: this.id }) }, {
+      relationships: { group },
+    });
+    clinician = this.toRelation(clinician);
+
+    this.set({ _clinicians: this.get('_clinicians').concat(clinician.data) }, {
+      relationships: { clinician },
+    });
+
+    $.ajax({
+      type: 'POST',
+      url,
+      data: JSON.stringify({ data: [clinician.data] }),
```

Comment:
does it expect the data to be in an array here and in the delete?  I would expect it's just looking for `{ id, type }` ?

Reply:
The API docs seem to indicate that the endpoints accept an array. But is it possible that it can accept either a single clinician or an array?

Reply:
yeah you're right.. that's surprising, but I guess it's easy enough to implement.

Reply:
I assumed it was to allow for bulk adding/removing.